### PR TITLE
chore(docs): consider bottom safe area for aside

### PR DESCRIPTION
**Pull request by yinonov**

### CODE REVIEW

File: apps/docs/assets/styles/main.css [hunk 1]
```diff
@@ -28,25 +29,22 @@ a:visited {
...
 }
 
-#menu-button:defined {
-  display: block;
-  position: fixed;
-  inset: auto auto 8px 8px;
-  z-index: 2;
+#menu-button:not(:defined) {
+	display: none;
 }
```
Comment (rachelbt) [inline apps/docs/assets/styles/main.css:40]: lines 31-40 all this is not needed anymore - the button is now in the header

File: apps/docs/assets/styles/main.css [hunk 2]
```diff
@@ -68,6 +66,7 @@ vwc-side-drawer::part(base) {
   block-size: calc(100vh - var(--_header-block-size));
   bottom: 0;
   top: var(--_header-block-size);
+	padding-block-end: calc(40px + env(safe-area-inset-bottom));
```
Comment (rachelbt) [inline apps/docs/assets/styles/main.css:69]:
```suggestion

```
this is not needed anymore - the fab button is not there anymore...

File: apps/docs/_layouts/base.njk [hunk 3]
```diff
@@ -4,7 +4,7 @@
   <head>
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+		<meta name='viewport' content='initial-scale=1, viewport-fit=cover'>
```
Comment (rachelbt) [inline apps/docs/_layouts/base.njk:7]: 👍

File: apps/docs/assets/styles/main.css (hunk 2 above)
Comment (yinonov) [reply apps/docs/assets/styles/main.css:54]:
```suggestion
	padding-block-end: env(safe-area-inset-bottom);
```

File: apps/docs/assets/styles/main.css [hunk 4]
```diff
@@ -14,9 +14,10 @@ html {
 }
 
 body {
-  margin: 0;
+	margin: 0;
   background-color: var(--vvd-color-canvas);
   color: var(--vvd-color-on-canvas);
+	direction: ltr;
```
Comment (yinonov) [inline apps/docs/assets/styles/main.css:20]:
```suggestion
  margin: 0;
  background-color: var(--vvd-color-canvas);
  color: var(--vvd-color-on-canvas);
```

**Pull request merged**